### PR TITLE
[Feat] Typography component

### PR DESCRIPTION
**Pull request by rushikesh3299**
Typography component with utility classes.
Missing semicolan in toast.css
https://deploy-preview-9--saraniui.netlify.app/components/typography/typography

### CODE REVIEW

File: Components/Typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                <i class="fas fa-bars" id="menu"></i>
+                <div class="nav-header fw900">सरणी UI</div>
+            </div>
+            <div class="nav-right">
+                <a href="https://github.com/rushikesh3299"><i class="fab fa-github"></i></a>
+                <a href="https://twitter.com/rushikesh3299"><i class="fab fa-twitter"></i></a>
+                <a href="https://www.linkedin.com/in/rushikesh-chougule-621854192/"><i class="fab fa-linkedin-in"></i></a>
+            </div>
+        </div>
+        <div class="side-bar pb40">
```
Comment (ShreyashMahajan) [inline Components/Typography/typography.html:25]: you can use aside container here

File: Components/Typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <iframe
+                    src="https://carbon.now.sh/embed/9myQNBlXzwITcH1Lq6Wk"
+                    style="width: 451px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <p class="component-specific">As shown below you can use utility calsses taC, taL & taR which means text-align to center, left or right respectively</p>
+            <div class="component-demo-box">
+                <div class="typography-box2">
+                    <p class="taC">Text Align Center</p><hr>
```
Comment (ShreyashMahajan) [inline Components/Typography/typography.html:67]: better naming convention can be used here

**Review by ShreyashMahajan — APPROVED** [review]
Just looks at the comments, Code is well structured. Good to merge

**Review by t007rushi — APPROVED** [review]
Looks good to merge just one thing you can use shadow/border for header to separate from main content

**Pull request merged**